"use strict";

try{
    x = 10;
} catch(error){
    console.log(error);
}finally{
    console.log("El programa continua");
}
